main.go

```diff
@@ -142,9 +145,15 @@ func main() {
 		glog.Info("Postgres metrics connection string was not set, postgres metrics are disabled.")
 	}
 
+	vodDecryptPrivateKey, err := crypto.LoadPrivateKey(cli.VodDecryptPrivateKey)
```

Comment:
We should probably make this logic optional so the startup doesn't crash if these new args are not provided. Just safer in general to avoid failed deploys etc. Could you surround all this logic with a check for whether the CLI args were provided?